### DISCUSSION TITLE
Protection for Frigate-Config access through Image-Entity

### DISCUSSION THREAD

**Discussion by albotroz** [blakeblackshear/frigate#9615] [Ideas]
With the latest version of Frigate, image entities have been introduced.

I have several people who have access to Dashboard via Home Assistant. To prevent them from being able to click and change anything else, I have hidden the side- & menubar etc. using an HA-Integration.
So they only see live streams from the cameras and the latest events.
In the previous Frigate version, I simply added the events to the dashboard via Frigate Cards. You couldn't click on them and change anything.

Now you can click on the last events and go further into Frigate and change settings unintentionally.

The possibility to lock access to it (outlined in orange) with a PIN would be desirable - if possible.
![image entity](https://github.com/blakeblackshear/frigate/assets/67508404/f4872049-5fb3-4088-8448-c56ea2180b3b)

**Comment by NickM-27** [comment]
image entities existed previously as camera entities, this is not a new feature and the same options were available prevoiusly.

also, pin protection would be a feature request for home assistant not frigate
